programs/ziti-prox-c/proxy.c

```diff
@@ -85,6 +85,8 @@ struct client {
     LIST_ENTRY(client) next;
 };
 
+uv_loop_t *global_loop;
```

Comment:
there some magic referencing this? is it used elsewhere? i don't see it anywhere else in the pr

Reply:
This is a test program.

Reply:
Yea I did a bad job of reverting. I moved the declaration up in the file for some debug code. I removed the duplicate.